Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.12-bullseye
+FROM python:3.9-bullseye
```

Comment:
Just wondering if you have tried newest version?  (Latest stable one is 3.12)

3.9 is also nice, it supports https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
Thanks for checking